### PR TITLE
Make Split Layouts Accept any number of children

### PR DESCRIPTION
**Pull request by SaraVieira**
Heey !

Adds that feauture and also added a new slide deck I made and vscode did some Prettier things on the Readme file. Let me know if I should revert it

### CODE REVIEW

File: src/layouts/Split.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ const Root = styled.div([], {
 })
 
 const Split = ({ children }) => {
-  const [ a, ...rest ] = React.Children.toArray(children.props.children)
+  const kids = React.Children.toArray(children.props.children)
```
[the remark this answers is not in the document]
Comment (SaraVieira) [reply src/layouts/Split.js]: This makes sense! Do you have an idea what to name the component?

[the remark this answers is not in the document]
Comment (jxnblk) [reply src/layouts/Split.js]: Good question... 🤔 Mind if I give it some thought over the next few days?

[the remark this answers is not in the document]
Comment (SaraVieira) [reply src/layouts/Split.js]: Ofc ! Ping me here or on twitter :)

[the remark this answers is not in the document]
Comment (jxnblk) [reply src/layouts/Split.js]: Not sure if I have a good idea of what this should be called or not, but does maybe `Horizontal` make sense?

[the remark this answers is not in the document]
Comment (SaraVieira) [reply src/layouts/Split.js]: Fixed :)

**Comment by SaraVieira** [comment]
Added example too :)

File: README.md [hunk 2]
```diff
@@ -320,6 +316,7 @@ See more exporting options in the [Exporting Documentation](docs/exporting.md)
 - [Bringing Brazil to the Cloud, Now][brazil-now] by [Guillermo Rauch](https://mobile.twitter.com/rauchg/)
 - [Simplify React][simplify-react] by [Kent C. Dodds](https://mobile.twitter.com/kentcdodds)
 - [I Got 99 Problems but GraphQL Ain't One][99-problems] by [Sara Vieira](https://mobile.twitter.com/NikkitaFTW)
+- [Stop de #divFest][stop-div-fest] by [Sara Vieira](https://mobile.twitter.com/NikkitaFTW)
```
Comment (jxnblk) [inline README.md]: 👌

**Comment by jxnblk** [comment]
Thanks for this and sorry for the delay! I'm going to merge this in and remove the yarn.lock file, then publish a release

**Pull request merged**